SQL/0000-00-00-schema.sql

```diff
@@ -225,16 +225,32 @@ CREATE TABLE `test_subgroups` (
 
 INSERT INTO test_subgroups (Subgroup_name) VALUES ('Instruments'),('Imaging');
 
+CREATE TABLE `instrument_schema` (
+  `InstrumentSchemaID` int(10) unsigned NOT NULL auto_increment,
+  `PreviousVersion` int(10) unsigned default NULL,
+  `UsersID` int(10) unsigned NOT NULL,
+  `SchemaHash` char(64) NOT NULL,
```

Comment:
What hash is this hashed with? It should have the hashing algorithm in the name so that it can be interpreted.